src/tactic/core.lean

```diff
@@ -456,6 +456,19 @@ do cxt ← list.map expr.to_implicit_local_const <$> local_context,
    add_decl $ declaration.defn n univ t' d' (reducibility_hints.regular 1 tt) trusted,
    applyc n
 
+/-- Replacement for `resolve_constant`. `resolve_constant` crashes
+when called on a name defined within a section that also uses `parameters`.
+This version handles can be used in all the same definitions as `resolve_constant`
+plus those declared within the scope of `parameters`. -/
+meta def resolve_constant' (n : name) : tactic name :=
+do e ← resolve_name n,
+   match e with
+   | expr.const n _ := pure n
+   | _ := do
+     e ← to_expr e,
```

Comment:
A think you want `ff tt` like in my version or this could fail for stupid reasons

Reply:
What do you mean by `tt` `ff`?

Reply:
Yes, ok I see